roles/inspec/templates/etc/inspec/host-controls/controls/control.rb

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
+# license: Apache 2.0
+# title '{{ item }} controls'
+
+
```

Comment:
I think we are missing a case where we just want to run all the controls and we don't skip any.

1. Required controls is non-empty: use `required_controls` and fill list
2. Skip controls is non-empty: use `include_controls` and fill list with skipped controls
3. Skip controls is empty: do `include_controls` with an empty list